packages/volto-iframe-block/news/7.feature

```diff
@@ -0,0 +1,4 @@
+move isValidUrl to helper function [@jnptk]
+add description and credit fields [@jnptk]
+improve resetBlock function [@jnptk]
+enhance View component [@jnptk]
```

Comment:
People read the release notes to find out if it's safe to upgrade. So it's best to be specific.

```suggestion
Move isValidUrl to helper function so it can be shadowed. [@jnptk]
Add description and credit fields. [@jnptk]
Clear all fields when the reset button is clicked. [@jnptk]
Show title, description, and credit in the View component. [@jnptk]
```